5.2.0/centos/Dockerfile

```diff
@@ -42,16 +42,23 @@ EXPOSE 4466
 RUN yum update -y && yum clean all
 
 RUN useradd --system idsvr
-USER idsvr:idsvr
 
 ENV IDSVR_HOME /opt/idsvr
 ENV JAVA_HOME $IDSVR_HOME/lib/java/jre
 ENV PATH $IDSVR_HOME/bin:$JAVA_HOME/bin:$PATH
 WORKDIR $IDSVR_HOME
 
-COPY --chown=idsvr idsvr-5.2.0/idsvr /opt/idsvr
-COPY --chown=idsvr first-run /opt/idsvr/etc/first-run
+COPY idsvr-5.2.0/idsvr /opt/idsvr
+COPY first-run /opt/idsvr/etc/first-run
 COPY --from=openssl-builder /build/lib/lib*so* /usr/lib64/
 COPY --from=openssl-builder /build/bin/openssl /bin/
 
+RUN find /opt/idsvr -type f -exec chmod a-w {} \; && \
```

Comment:
why do this OpenShift thing for CentOS as well. It was only in Ubuntu systems before so we should not chgrp the files to root for this distro.

Reply:
Thought it could be nice to have in all places but now reverted changes except ubuntu. 
Learning things PR to PR 👍

Reply:
Learning on the job is the way it goes, so NP, @ambatimuralikrishna .